src/components/ReaactCompoents/WikiPost/WikiPost.tsx

```
@@ -0,0 +1,204 @@
+
+  const [selectedTag, setSelectedTag] = useState<string | null>(null);
+  const [currentPage, setCurrentPage] = useState(1);
+  const versionsPerPage = 6;
+
+  const sortedVersions = useMemo(() => {
+    try {
+      return mappedVersions.sort((a, b) => {
+        // Handle semantic versioning (e.g., "1.2.3")
+        const parseVersion = (v: string) => {
+          const parts = v.split(".").map(Number);
+          if (parts.some(isNaN)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)